tests/sanity/ignore-2.10.txt

```diff
@@ -17,6 +17,7 @@ tests/integration/targets/helm/files/appversionless-chart-v2/templates/configmap
 tests/integration/targets/helm/files/appversionless-chart/templates/configmap.yaml yamllint!skip
 tests/integration/targets/helm/files/test-chart-v2/templates/configmap.yaml yamllint!skip
 tests/integration/targets/helm/files/test-chart/templates/configmap.yaml yamllint!skip
+tests/integration/targets/helm_diff/files/test-chart/templates/configmap.yaml yamllint!skip
```

Comment:
Can you add these changes to the sanity refresh script? Otherwise they will get lost the next time that script is run. That script should be the only way changes get made to these ignore files.